actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -11,40 +9,25 @@ module CollectionCaching # :nodoc:
     end
 
     private
-      def cache_collection_render
-        return yield unless cache_collection?
+      def cache_collection_render(instrumentation_payload)
+        return yield unless @options[:cached]
```

Comment:
out of curiosity, what was the rationale of going from `@options[:cache]` to `@options[:cached]`?

this bit us when we upgraded to rails 5 and removed `multi_fetch_fragments` from our Gemfile - it took ages to figure out why collection caching just wasn't happening anymore.